dockerfile_scripts/install_deb_packages.sh

```diff
@@ -0,0 +1,37 @@
+apt-get update \
+	&& DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		autoconf \
+		automake \
+		autotools-dev \
+		build-essential \
+		ca-certificates \
+		curl \
+		daemontools \
+		debhelper \
+		devscripts \
+		ibverbs-providers \
+		libibverbs1 \
+		libkrb5-dev \
+		librdmacm1 \
+		libssl-dev \
+		libtool \
+		git \
+		krb5-user \
+		g++ \
+		cmake \
+		make \
+		openssh-client \
+		openssh-server \
+		pkg-config \
+		wget \
+		nfs-common \
+                libnuma1 \
+                libnuma-dev \
+                libpmi2-0-dev \
+		unattended-upgrades \
+	&& unattended-upgrade \
+	&& rm -rf /var/lib/apt/lists/* \
+	&& rm /etc/ssh/ssh_host_ecdsa_key \
+	&& rm /etc/ssh/ssh_host_ed25519_key \
+	&& rm /etc/ssh/ssh_host_rsa_key \
+   && ln -s /usr/include/slurm-wlm /usr/include/slurm
```

Comment:
this last line looks hpc-specific, should be be put in the *-hpc images instead?

Reply:
Also, I noticed that in Dockerfile-base-gpu, this is behind a version switch, do you know what that's about?

Reply:
from https://github.com/determined-ai/environments/pull/156,
```
# We need to create sym links for the Slurm PMI headers if we are using 
# Ubuntu 18.04 because they are not installed in a standard location.
```

Reply:
This line can just be removed entirely since we don't use Ubuntu 18.04 anymore

Reply:
I think we can remove this since we don't build with Ubuntu 18.04 anymore.